akka-javasdk-tests/src/test/java/akkajavasdk/components/workflowentities/TransferWorkflow.java

```diff
@@ -99,7 +109,23 @@ public Effect<Message> getLastStep() {
     return effects().reply(new Message(currentState().lastStep()));
   }
 
-  public record SomeClass(String someValue) {}
+  public Effect<TransferState> get() {
+    if (currentState() == null && isDeleted()) {
+      return effects().reply(TransferState.EMPTY);
+    }
+    return effects().reply(currentState());
+  }
+
+  public Effect<Done> delete() {
+    return effects().delete().thenReply(done());
+  }
+
+  public Effect<Boolean> getIsDeleted() {
```

Comment:
`getIs` sounds funny, why not `isDeleted`?

Reply:
ah, there is already `isDeleted` from the base class. what about `hasBeenDeleted` ?

Reply:
much better